configurations/gcp/user-agent-config.yml

```diff
@@ -0,0 +1,4 @@
+---
+- type: replace
+  path: /cloud_provider/properties/google/user_agent_prefix?
+  value: "kubo/0.0.3"
```

Comment:
Yea, it's not ideal.. It's a little weird because this is referencing the release number, but the release number is actually stored and updated in kubo-release.